db/seeds.rb

```diff
@@ -1,56 +1,222 @@
 # frozen_string_literal: true
 
-require 'csv'
+firestone = Building.find_or_create_by(name: 'Firestone Library', email: 'access@princeton.edu')
+lewis = Building.find_or_create_by(name: 'Lewis Library', email: 'lewislib@princeton.edu')
+Rails.logger.warn("Created #{Building.count} Buildings")
 
-Building.seed
+# Setup flipflop for feature swap - borrowed from https://github.com/voormedia/flipflop/blob/ed1fd56f2f86974004c7af819612dd1622adcf33/lib/tasks/flipflop.rake
+# For some reasone when we use the rake tasks directly, only the first one has an impact
```

Comment:
I ran into this recently too.  `#invoke` [only runs if it feels that it is "needed"](https://www.rubydoc.info/gems/rake/Rake%2FTask:invoke)

You can, however, run `#reenable` (and, if needed, `#clear`) like so: https://github.com/pulibrary/lockers_and_study_spaces/blob/8548f8156289ae575d87e31b7dd85c8b914f9325/spec/lib/tasks/lockers_rake_spec.rb#L59-L61

If the rake task doesn't depend on others, I think you can also simply call `#execute` instead of `#invoke`

Reply:
Ah, good info to have! Thank you!